src/pages/HomePage/components/Comments/Comments.tsx

```diff
@@ -1,9 +1,91 @@
 import './Comments.scss';
+import rightarrow from  '../Comments/CommentImages/arrow right.svg';
+import quote from  '../Comments/CommentImages/Vector.svg';
+import line from  '../Comments/CommentImages/Line 6.svg';
+
 
 export const Comments= () =>{
     return(
-        <>
-        <h1>Comments</h1>
-        </>
+        <div className="comments narrow-container">
+    <div className="mixin">
+       <p className="explanation">We cook – they enjoy every <br/> minute of their <img  className="rightarrow"  src={rightarrow}/>
+        lives</p>
+        <p className="because">Because we...</p>
+      <div className="quote">
+          <img src={quote}/>
+          <p className="quotation">We don't accept half-measures and  shortcuts, because what we care <br/> about  is the WOW effect. And all the 
+          luck that our efforts are not in <br/>  vain. We know this because we closely follow the opinions of our <br/>  consumers.</p>
+        </div> 
+    </div>
+ <div className="fourpart"> 
+    <div className="comment-box">
```

Comment:
you must create a Comment component and move this html inside of this component

in Comments component you will have an array of comments, and you've got to use comments.map(...) to render your Comment components